tools/cli/include/nearobject/cli/NearObjectCliHandler.hxx

```diff
@@ -97,6 +98,7 @@ struct NearObjectCliHandler
 
 private:
     NearObjectCli* m_parent;
+    std::shared_ptr<::uwb::UwbSession> m_activeSession{ nullptr };

```

Comment:
For default constructor I'm pretty sure we can just say
```suggestion
std::shared_ptr<::uwb::UwbSession> m_activeSession;
```

Reply:
Yep, done.